samza-rest/src/main/java/org/apache/samza/rest/script/ScriptRunner.java

```diff
@@ -126,4 +153,34 @@ public void run() {
     log.debug("Exit value {}", exitVal);
     return exitVal;
   }
+
+  /**
+   * @return true if this runner will forward the current environment variables to the child process, false otherwise.
+   */
+  public boolean forwardEnvironmentVars() {
+    return forwardEnvironmentVars;
+  }
+
+  /**
+   * Set the flag indicating whether this runner will forward the current environment variables to the child process.
+   *
+   * @param forwardEnvironmentVars  true to forward the current environment to the child process,
+   *                                false to start with an empty environment.
+   */
+  public void setForwardEnvironmentVars(boolean forwardEnvironmentVars) {
```

Comment:
A couple of alternative approaches 

- Do we need this setter? Can get rid off this to ensure that the `forwardEnvVars` is initialized during construction.

- Alternately, it feels natural to add `forwardEnvVars` to the `runScript` API since it's designed to run a script with IO inherited from the current Java process anyway.